content-src/components/VisitAgain/VisitAgain.js

```diff
@@ -0,0 +1,72 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {justDispatch} = require("common/selectors/selectors");
+const {actions} = require("common/action-manager");
+const {SpotlightItem} = require("components/Spotlight/Spotlight");
+const CollapsibleSection = require("components/CollapsibleSection/CollapsibleSection");
+const getBestImage = require("common/getBestImage");
+const {VISITAGAIN_DISPLAYED_LENGTH} = require("common/constants");
+
+const VisitAgain = React.createClass({
+  getDefaultProps() {
+    return {
+      length: VISITAGAIN_DISPLAYED_LENGTH,
+      page: "NEW_TAB",
+      placeholder: false
+    };
+  },
+  onClickFactory(index, site) {
+    return () => {
+      let payload = {
+        event: "CLICK",
+        page: this.props.page,
+        source: "VISITAGAIN",
+        action_position: index,
+        highlight_type: site.type,
+        metadata_source: site.metadata_source
+      };
+      this.props.dispatch(actions.NotifyEvent(payload));
+    };
+  },
+  // XXX factor out into a stateless component
```

Comment:
I know this is copied from the other component, do you know what it means? 😕 This component doesnt seem to use `this.state`, so not sure.